hack/testing/logging.sh

```diff
@@ -421,7 +421,7 @@ if [ "$TEST_PERF" = "true" ] ; then
     echo "Running performance tests"
     for test in perf-*.sh ; do
         if [ -x ./$test ] ; then
-            ./$test $USE_CLUSTER
+            (. ./$test $USE_CLUSTER)
```

Comment:
If you are sourcing the file it doesn't have to be executable, but I guess the `-x` test is useful to know which files are meant to be run.